src/components/FormInputs/FormInputs.jsx

```diff
@@ -0,0 +1,34 @@
+import * as S from './formInputs.styles';
+
+const FormInputs = ({ name, onNameChange, description, onDescriptionChange, price, onPriceChange }) => {
+  return (
+    <>
+      <label>상품명</label>
+      <input
+        css={S.inputStyle}
+        placeholder="상품명을 입력해주세요"
+        value={name}
+        onChange={(e) => onNameChange(e.target.value)}
+      />
+
+      <label>상품 소개</label>
+      <textarea
```

Comment:
💊 제안
label 태그 사용하신 것 좋아요! input과 연결도 시켜주시면 더 좋을 것 같아요.

```suggestion
      <label htmlFor="id">상품 소개</label>
      <textarea
        id="id"
```